mon-pix/app/components/training/card.hbs

```diff
@@ -0,0 +1,18 @@
+<a class="training-card" href={{@training.link}} target="_blank" rel="noopener noreferrer">
+  <picture class="training-card__visual">
+    <img
+      src="{{this.rootURL}}/images/logo/logo-ministere-education-nationale-et-jeunesse.svg"
```

Comment:
C'est vraiment utile le `{{this.rootURL}}` ?

Reply:
Aucune idée 🤷🏻‍♂️
J'ai repris ça d'une autre image statique importée dans l'app.

Reply:
Je vois que les 2 façons sont présentes, d'après la doc ce n'est pas nécessaire : https://cli.emberjs.com/release/advanced-use/asset-compilation/#rawassets

Reply:
Modifié